### DISCUSSION TITLE
Restart, Interrupt buttom

### DISCUSSION THREAD

**Discussion by localcreator** [microsoft/vscode-jupyter#10759] [Questions and Answers]
![112233](https://user-images.githubusercontent.com/76257407/178240181-fa66b01a-5912-4625-955a-9c24fdc1fd2a.png)
Where did the Restart, Interrupt buttons disappear from the navigation bar in version 1.69.0?

**Comment by IanMatthewHuff** [comment]
@oscar-defelice Thanks for reporting this and sorry about the issue. I believe that we have a localization issue hitting some users currently. Looking into it.
